content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -847,6 +847,41 @@ not vulnerable to ordering changes in the list.
 
 Once the last finalizer is removed, the resource is actually removed from etcd.
 
+### Force deletion
```

Comment:
(nit)
```suggestion
### Forcible resource deletion {#forced-deletion}
```

Reply:
should be `{#force-deletion}`, no `d`, to preserve existing links.

Reply:
@tengqm this is new content (no existing links). Here I'm just recommending a way to make the eventual URL shorter.